src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/general/_mixins.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ Allow rem units with px fallback
+*/
+
+$rem-baseline: $font-size-normal !default;
+$rem: true !default;
+$rem-px-only: false !default;
+
+// list-separator polyfill by Hugo Giraudel (https://sass-compatibility.github.io/#list_separator_function)
+@function rem-separator($list) {
+    @if function-exists("list-separator") == true {
+        @return list-separator($list);
+    }
+
+    $test-list: ();
+    @each $item in $list {
+        $test-list: append($test-list, $item, space);
+    }
+
+    @return if($test-list == $list, space, comma);
+}
+
+@mixin rem-baseline($zoom: 100%) {
+    font-size: $zoom / 16px * $rem-baseline;
+}
+
+@function rem-convert($to, $values...) {
+    $result: ();
+    $separator: rem-separator($values);
+
+    @each $value in $values {
+        @if type-of($value) == "number" and unit($value) == "rem" and $to == "px" {
+            $result: append($result, $value / 1rem * $rem-baseline, $separator);
+        } @else if type-of($value) == "number" and unit($value) == "px" and $to == "rem" {
+            $result: append($result, $value / ($rem-baseline / 1rem), $separator);
+        } @else if type-of($value) == "list" {
+            $result: append($result, rem-convert($to, $value...), $separator);
+        } @else {
+            $result: append($result, $value, $separator);
+        }
```

Comment:
Just to improve readability I would set a variable `$item` in the if-clause and have `$result: append($result, $item, $separator);` in a separate line.